db/migrate/20150912205745_create_seed_trades.rb

```diff
@@ -0,0 +1,17 @@
+class CreateSeedTrades < ActiveRecord::Migration
+  def change
+    create_table :seed_trades do |t|
+      t.text       :message
+      t.text       :address
+      t.datetime   :requested_date
+      t.datetime   :accepted_date
+      t.datetime   :declined_date
```

Comment:
Is there a reason for having two fields here, or should we have "replied_date" and a boolean "reply/answer" for accepted or declined? In theory any one record will only have either accepted or declined, not both, and that would mean we just need one index on replied_date.